portfolio/src/main/webapp/index.html

```diff
@@ -1,52 +1,112 @@
-              alt="Github logo"></img></a>
+    <!--Top bar-->
+    <ul class="nav nav-tabs">
+      <li class="active"><a data-toggle="pill" href="#header">Home</a></li>
+      <li><a data-toggle="pill" href="#cv">Resume</a></li>
+      <li><a data-toggle="pill" href="#entertainment">Entertainment</a></li>
+    </ul>
+
+    <div class="tab-content">
+      <!--Home page-->
+      <div class="card tab-pane fade in active" id="header">
+        <h1 id="title">Domene's Portfolio<br></h1>
+        <img id="titleimg" src="/images/me.jpeg" alt="Portfolio picture of Javier Domene"></img>
+        <p>This is my portfolio.</p>
+        <p>I am a second year student at Tec de Monterrey studying computer science.</p>
```

Comment:
I'd omit "second year" and just say "student". Otherwise, really easy to forget to update this to "third year" next year.